src/rules/orderedImportsRule.ts

```diff
@@ -51,16 +60,23 @@ export class Rule extends Lint.Rules.AbstractRule {
                     type: "string",
                     enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
                 },
+                "source-imports-order": {
+                    type: "string",
+                    enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
+                },
             },
             additionalProperties: false,
         },
-        optionExamples: ["true", '[true, {"named-imports-order": "lowercase-first"}]'],
+        optionExamples: [
+            "true",
+            '[true, {"named-imports-order": "lowercase-first", "source-imports-order": "lowercase-last"}]',
```

Comment:
is it actually useful to set a different option for the import names vs. the import sources? I guess if we wanted to consolidate those options into one, it might be a breaking change... but this feels a bit ugly